src/lazy/decoder.rs

```diff
@@ -395,47 +411,44 @@ pub(crate) mod private {
     }
 
     pub trait LazyRawStructPrivate<'top, D: Decoder> {
-        /// Creates an iterator that converts each raw struct field into an `UnexpandedField`, a
+        /// Creates an iterator that converts each raw struct field into an `FieldExpr`, a
         /// common representation for both raw fields and template fields that is used in the
         /// expansion process.
-        fn unexpanded_fields(
+        fn field_exprs(
```

Comment:
🪧 `UnexpandedField` was a clunky name, hopefully this is better. The new relationship between `LazyExpandedField` and `FieldExpr` mirrors the existing relationship between `LazyExpandedValue` and `ValueExpr`.